tests/functional/monitoring/prometheus/metrics/test_ocs_exporter_deployment.py

```
@@ -0,0 +1,292 @@
+                resource_name="ocs-metrics-exporter",
+            )
+            sm_data = ocp_sm.get()
+            endpoints = sm_data.get("spec", {}).get("endpoints", [])
+            for endpoint in endpoints:
+                ep_port = endpoint.get("port", "")
+                assert "proxy" not in ep_port.lower(), (
+                    f"ServiceMonitor should not reference proxy port, "
+                    f"got port={ep_port}"
+                )
+            logger.info(
+                "Check 5 PASSED: ServiceMonitor targets exporter port " "directly"
```

Ruff (0.16.0)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)